server/models/Post.js

```diff
@@ -56,6 +61,7 @@ class Post {
                                                     FROM posts
                                                     WHERE posts.unique_id = ${special_id}`);
                 let post = new Post(result.rows[0])
+                console.log(post)
                 res(post)
```

Comment:
- Nice flow in your functions (indentation, commenting etc)
- This file (models/posts) is different to what we seemed to have cloned down - not sure if we did something wrong here.